ts/data/test.ts

```diff
@@ -0,0 +1,371 @@
+import { legendAction, TableDefinition } from "../component/table-builder";
+import {
+  addLink,
+  makeIcon,
+  styleText,
+  addMisoIcon,
+  makeNameDiv,
+  addNaText,
+  addTextDiv,
+  makeTextDivWithTooltip,
+} from "../util/html-utils";
+import { siteConfig } from "../util/site-config";
+import { urls } from "../util/urls";
+import {
+  Test,
+  Donor,
+  Project,
+  makeSampleTooltip,
+  samplePhaseComplete,
+  addConstructionIcon,
+  addSpace,
+  assertNotNull,
+  samplePhasePendingWork,
+} from "./case";
+import { Assay } from "./assay";
+import { getQcStatus, Sample } from "./sample";
+import { QcStatus, qcStatuses } from "./qc-status";
+import { Requisition } from "./requisition";
+import { Tooltip } from "../component/tooltip";
+import { caseFilters, latestActivitySort } from "../component/table-components";
+
+export interface TestTableView {
+  test: Test;
+  caseId: string;
+  requisition: Requisition;
+  donor: Donor;
+  projects: Project[];
+  assay: Assay;
+  tissueOrigin: string;
+  tissueType: string;
+  timepoint: string;
+  receipts: Sample[];
+}
+
+export const testDefinition: TableDefinition<TestTableView, void> = {
+  queryUrl: urls.rest.tests,
+  defaultSort: {
+    columnTitle: "Test",
+    descending: true,
+    type: "text",
+  },
```

Comment:
I think we need to add a "Latest Activity" column to use for sorting because that will be much more useful, and should be the default sort (descending)